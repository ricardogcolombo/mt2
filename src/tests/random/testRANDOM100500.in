100 500
1 34 35 27 59
1 26 25 63 71
1 44 67 7 66
1 21 34 66 122
1 37 119 42 23
1 13 43 66 108
1 81 96 99 88
1 23 71 16 5
1 69 123 50 88
1 66 49 20 7
1 77 101 68 27
1 4 125 6 94
1 35 108 34 44
1 56 109 3 47
1 98 15 12 102
1 9 113 50 101
1 32 75 71 8
1 60 34 93 3
1 67 101 34 59
1 4 69 97 90
1 33 3 81 127
1 72 83 68 23
1 23 12 79 80
1 27 15 99 120
1 29 15 32 64
1 58 9 63 69
1 4 90 21 52
1 99 43 27 76
1 85 100 38 8
1 94 39 75 130
1 15 103 28 1
1 15 36 71 54
1 40 79 79 42
1 51 2 5 122
1 73 17 44 41
1 4 114 56 7
1 91 22 45 109
1 82 120 85 38
1 67 82 71 33
1 67 81 82 83
1 25 6 11 90
1 63 3 12 12
1 64 47 79 100
1 49 120 4 88
1 94 72 13 103
1 1 19 53 108
1 31 86 14 102
1 89 99 13 102
1 15 119 58 48
1 78 98 55 58
1 34 97 82 32
1 33 11 55 72
1 55 120 48 62
1 96 77 55 67
1 31 5 94 32
1 5 104 91 4
1 44 105 22 27
1 4 35 51 20
1 65 44 96 15
1 4 79 80 52
1 43 59 1 8
1 89 48 58 129
1 96 48 71 55
1 14 115 49 95
1 53 116 74 39
1 65 29 32 25
1 3 11 62 96
1 74 78 31 80
1 45 37 77 66
1 3 26 53 33
1 49 118 39 89
1 58 22 23 99
1 9 89 52 45
1 22 56 68 10
1 86 45 79 0
1 22 17 27 87
1 35 12 28 30
1 4 48 31 104
1 51 41 69 8
1 28 53 25 51
1 39 80 89 61
1 15 2 70 83
1 94 107 49 110
1 69 117 42 36
1 34 127 80 118
1 18 10 25 96
1 45 8 80 129
1 1 122 80 16
1 48 83 1 18
1 82 78 11 57
1 74 85 88 70
1 30 22 40 27
1 2 102 64 123
1 66 31 28 62
1 1 91 17 126
1 11 105 79 53
1 28 29 71 82
1 16 29 5 54
1 38 81 74 65
1 19 69 48 97
1 87 24 90 32
1 19 69 5 112
1 75 52 92 63
1 59 106 88 122
1 33 19 39 74
1 88 72 1 68
1 44 56 6 122
1 99 69 72 80
1 45 115 64 47
1 31 129 40 118
1 2 32 96 112
1 1 11 72 10
1 51 44 7 83
1 83 15 24 65
1 80 9 46 89
1 100 104 73 111
1 52 89 37 30
1 71 5 76 71
1 47 129 80 38
1 86 69 23 65
1 53 53 87 49
1 95 9 4 103
1 41 70 53 24
1 76 9 53 104
1 72 59 55 87
1 92 22 42 27
1 42 5 4 16
1 87 20 61 128
1 21 51 66 7
1 32 15 98 25
1 63 127 100 130
1 42 93 11 74
1 60 85 52 83
1 45 24 28 122
1 18 105 84 123
1 75 24 31 110
1 80 119 32 70
1 6 130 25 105
1 39 11 18 38
1 3 6 47 37
1 8 108 44 93
1 25 77 77 70
1 15 14 85 6
1 73 47 22 105
1 1 64 74 56
1 55 15 74 2
1 46 116 37 33
1 25 108 10 106
1 10 64 78 120
1 34 117 60 119
1 48 61 78 43
1 21 62 80 7
1 14 66 66 93
1 60 83 45 115
1 53 81 78 28
1 35 79 44 51
1 75 35 14 60
1 5 90 27 81
1 29 100 46 69
1 12 17 75 76
1 56 91 38 116
1 7 42 44 57
1 2 0 64 36
1 65 127 64 31
1 53 11 35 36
1 63 129 55 81
1 88 115 84 88
1 8 99 23 76
1 12 104 22 128
1 22 41 98 83
1 40 96 19 4
1 13 33 80 62
1 50 97 91 126
1 63 37 64 124
1 64 8 36 92
1 89 30 16 58
1 46 93 47 114
1 42 121 17 105
1 49 63 84 55
1 69 121 52 92
1 35 101 55 61
1 68 64 47 50
1 87 117 62 24
1 74 1 4 42
1 82 22 99 38
1 36 105 89 17
1 16 109 42 59
1 19 67 84 75
1 18 19 5 40
1 33 19 20 8
1 27 102 29 96
1 53 88 57 96
1 11 19 46 120
1 18 22 76 2
1 46 23 98 21
1 15 63 11 102
1 46 97 2 123
1 31 100 82 122
1 74 85 56 83
1 34 9 92 81
1 80 108 63 34
1 95 82 10 99
1 98 104 74 7
1 69 27 95 40
1 29 122 40 82
1 45 26 41 23
1 88 130 93 48
1 39 54 91 76
1 23 112 6 34
1 17 18 1 44
1 93 27 95 40
1 13 43 30 118
1 14 27 92 124
1 74 84 87 14
1 32 58 21 14
1 36 104 35 83
1 59 119 45 95
1 73 46 35 83
1 94 112 65 23
1 4 38 98 106
1 20 81 65 62
1 64 127 87 21
1 75 75 42 102
1 52 98 58 21
1 39 9 5 110
1 73 9 40 71
1 78 17 88 1
1 63 66 11 116
1 15 53 36 59
1 63 85 38 77
1 62 112 67 90
1 51 86 27 102
1 94 10 98 9
1 67 31 33 105
1 22 91 71 33
1 8 130 81 101
1 95 17 64 45
1 77 62 96 32
1 68 5 99 35
1 99 39 18 81
1 65 30 12 109
1 53 90 64 48
1 2 91 10 128
1 15 73 4 96
1 13 127 6 112
1 50 82 73 54
1 36 109 7 7
1 89 3 42 47
1 65 124 81 125
1 15 7 49 111
1 88 66 67 86
1 20 103 99 96
1 53 18 97 51
1 50 93 45 4
1 56 101 19 12
1 89 108 88 29
1 87 14 57 98
1 54 5 4 96
1 74 35 2 77
1 45 113 86 25
1 1 48 12 89
1 46 107 90 18
1 91 106 47 58
1 33 127 47 109
1 43 44 62 113
1 79 17 31 106
1 77 31 81 33
1 5 3 54 89
1 41 52 28 104
1 45 115 74 59
1 82 60 18 78
1 3 121 86 59
1 41 14 57 115
1 33 66 29 67
1 86 30 79 84
1 97 18 35 31
1 96 24 25 5
1 75 10 52 121
1 20 34 37 107
1 32 58 95 76
1 53 23 55 52
1 72 129 100 11
1 86 98 48 94
1 1 102 82 113
1 93 38 73 95
1 30 116 52 77
1 95 93 39 86
1 46 25 55 20
1 57 85 84 117
1 38 1 28 8
1 13 71 87 32
1 51 10 34 48
1 75 71 37 115
1 18 99 47 43
1 6 106 93 65
1 37 74 72 80
1 53 36 42 126
1 12 60 76 105
1 95 45 6 87
1 59 26 68 112
1 64 110 76 20
1 41 123 100 41
1 52 39 68 88
1 80 73 2 70
1 90 115 47 60
1 69 69 86 35
1 98 17 38 111
1 90 79 31 102
1 46 43 1 25
1 14 53 56 75
1 62 99 74 87
1 70 81 5 108
1 2 48 54 86
1 87 102 83 3
1 53 77 61 103
1 91 66 68 13
1 11 1 30 46
1 90 69 91 29
1 88 52 61 9
1 93 58 45 90
1 85 49 93 34
1 29 110 100 86
1 66 43 78 58
1 2 113 82 6
1 78 12 23 113
1 3 24 81 73
1 49 32 54 39
1 10 92 57 20
1 26 107 82 4
1 87 38 83 57
1 52 76 71 83
1 90 79 80 128
1 2 112 9 106
1 75 120 35 56
1 75 130 28 15
1 57 34 49 21
1 89 83 56 62
1 93 53 94 72
1 91 92 29 7
1 44 81 11 83
1 12 122 100 39
1 34 129 5 35
1 67 115 33 88
1 99 82 67 66
1 24 69 67 54
1 6 33 33 121
1 54 88 35 50
1 82 54 75 78
1 80 11 79 30
1 19 1 99 4
1 69 83 61 100
1 22 30 24 102
1 16 77 10 95
1 32 51 23 116
1 28 20 13 35
1 28 59 50 47
1 66 69 75 112
1 75 114 80 7
1 84 62 90 46
1 33 5 95 82
1 2 23 34 96
1 59 79 84 113
1 23 9 92 85
1 5 52 12 93
1 29 14 78 124
1 19 57 51 60
1 19 28 53 95
1 49 75 31 122
1 99 80 72 123
1 30 2 67 89
1 75 123 10 88
1 69 22 75 126
1 13 62 33 48
1 78 25 13 76
1 86 73 66 115
1 11 81 59 118
1 66 107 9 46
1 22 104 5 109
1 43 18 21 31
1 16 122 49 0
1 77 15 36 46
1 93 10 67 74
1 55 57 60 81
1 20 63 87 42
1 19 109 67 22
1 27 99 19 49
1 14 1 18 3
1 35 67 13 117
1 1 38 49 60
1 52 41 50 123
1 61 56 80 43
1 94 80 39 117
1 60 53 72 121
1 73 25 66 2
1 50 87 32 109
1 81 44 39 102
1 75 106 83 112
1 81 71 48 80
1 65 5 54 66
1 73 27 10 88
1 31 36 57 130
1 99 79 1 94
1 72 85 73 29
1 74 126 95 66
1 72 87 71 92
1 66 59 81 118
1 96 116 92 82
1 32 1 9 38
1 28 64 1 6
1 56 23 85 88
1 10 40 8 1
1 48 54 16 65
1 13 33 47 119
1 78 9 80 105
1 48 74 34 23
1 11 128 43 78
1 61 24 37 7
1 51 66 92 113
1 69 8 60 28
1 43 60 18 24
1 84 72 24 54
1 19 29 67 121
1 95 114 69 62
1 55 28 18 100
1 14 16 25 102
1 82 114 91 10
1 92 102 40 96
1 95 113 87 37
1 92 50 72 59
1 6 25 17 52
1 67 62 16 130
1 80 92 36 25
1 28 105 96 57
1 69 124 57 0
1 99 40 22 21
1 4 76 51 49
1 64 128 65 69
1 60 103 43 129
1 56 50 94 30
1 28 32 99 122
1 53 78 23 35
1 15 100 5 20
1 14 74 69 99
1 40 55 90 126
1 45 108 80 114
1 39 79 45 8
1 54 112 22 10
1 42 27 82 22
1 3 7 43 127
1 64 123 69 39
1 8 18 53 127
1 85 24 6 61
1 13 4 56 26
1 100 82 99 3
1 10 57 44 4
1 79 84 37 62
1 30 59 60 85
1 88 119 66 55
1 73 69 90 21
1 35 48 94 74
1 42 92 11 42
1 14 4 93 65
1 51 68 65 95
1 54 32 50 125
1 19 112 85 6
1 19 15 58 115
1 98 54 60 111
1 2 76 62 80
1 96 8 63 70
1 97 113 76 48
1 92 35 32 57
1 39 42 13 38
1 88 4 76 112
1 5 92 25 18
1 24 11 57 12
1 97 92 36 74
1 70 9 81 80
1 4 130 86 100
1 98 85 72 64
1 97 94 89 79
1 7 82 44 30
1 25 1 49 14
1 93 90 38 52
1 93 70 58 88
1 22 127 91 69
1 17 106 7 14
1 87 10 56 115
1 59 113 7 42
1 80 99 20 13
1 37 39 22 30
1 60 70 90 96
1 71 106 69 91
1 81 1 16 49
1 57 38 76 13
1 24 75 85 114
1 42 52 54 11
1 77 38 18 114
1 100 82 35 71
1 61 103 62 26
1 23 54 29 77
