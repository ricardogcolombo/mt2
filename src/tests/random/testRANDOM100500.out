0.583333
0.500000
0.666667
0.625000
0.666667
0.400000
0.692308
0.583333
0.400000
0.444444
0.300000
0.714286
0.333333
0.500000
0.363636
0.500000
0.583333
0.529412
0.454545
0.500000
0.800000
0.727273
0.333333
0.444444
0.583333
0.769231
0.200000
0.714286
0.583333
0.200000
0.444444
0.250000
0.600000
0.400000
0.555556
0.666667
0.384615
0.333333
0.375000
0.545455
0.571429
0.400000
0.615385
0.333333
0.533333
0.166667
0.400000
0.750000
0.444444
0.714286
0.142857
0.454545
0.500000
0.636364
0.200000
0.500000
0.400000
0.692308
0.500000
0.500000
0.384615
0.545455
0.375000
0.416667
0.750000
0.375000
0.500000
0.625000
0.736842
0.800000
0.400000
0.400000
0.428571
0.500000
0.375000
0.444444
0.888889
0.333333
0.555556
0.333333
0.545455
0.562500
0.583333
0.700000
0.416667
0.285714
0.636364
0.250000
0.666667
0.500000
0.750000
0.428571
0.545455
0.500000
0.307692
0.533333
0.222222
0.571429
0.416667
0.500000
